### ISSUE TITLE
How to have local git instead of github?

### ISSUE THREAD

**Issue by Jiro90TC** [luciddreamz/laravel-ex#4]
How do I update my project from my local pc instead of github?
I were able to do that on openshift v2 but I can't understand how the v3 image work. 
Can you provide a simple guide on this?

**Comment by sweshgit** [comment]
Yes, please. I am also trying to do this. In OpenShift V2 I was able to localize the code and then push it to my application. But in OpenShift V3, I am supposed to push codes to Github first then to application.

**Comment by Jiro90TC** [comment]
Is there any solution on this? Openshift going to shutdown v2 and I need to upgrade to v3 by this month
